### PR TITLE
Patch/IP address date

### PR DESCRIPTION
**Pull request by britt6612**
[Deploy Preview](https://deploy-preview-INSERT_PR_#_HERE--keen-mayer-a86c8b.netlify.app/)

#### What does this PR do?
This PR Fixes `date` to be time stamp
also fixes IP address to appear to be real

**First Example**
- [x] Created On should be a date-time string
- [x] Format IP Address correctly

**Horizontal**
- [x] Format date-time
- [x] Format IP Address
- [x] Update description "Full-service A.I. insights, security and unified infrastructure management for campus, branch, remote, and data center networks."

**Vertical Example**
- [x] Created On should be a date-time string
- [x] Format IP Address correctly

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [ ] Console is free of warnings and errors
- [ ] Passes E2E commit checks
- [ ] Visual snapshots are reasonable

### CODE REVIEW

**Review by halocline — APPROVED** [review]

File: aries-site/src/examples/components/namevaluelist/Examples/NameValueListPairPropsExample.js [hunk 1]
```diff
@@ -9,7 +9,10 @@ export const NameValueListPairPropsExample = () => (
         let date;
         if (name === 'Created on') {
           const event = new Date(value);
-          date = event.toLocaleDateString();
+          date = event.toLocaleString('en-US', {
+            dateStyle: 'short',
```
Comment (taysea) [inline aries-site/src/examples/components/namevaluelist/Examples/NameValueListPairPropsExample.js:13]: I've heard comments from Edgar before that Greenlake tries to "zero pad" dates (I'm not sure about times) to make scanning multiple lines easier. So, for example 01/23/21, 7:15 AM. Should we do that? @halocline

Comment (britt6612) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListPairPropsExample.js:13]: `10/5/21, 8:48 AM`
This is what the preview is currently showing so is that not what this is  01/23/21, 7:15 AM?

Comment (taysea) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListPairPropsExample.js:13]: I think we can leave it to separate work handled here: https://github.com/grommet/hpe-design-system/issues/2039

Comment (britt6612) [reply aries-site/src/examples/components/namevaluelist/Examples/NameValueListPairPropsExample.js:13]: got it I was a little confused

**Pull request merged**